pkg/virt-operator/resource/generate/install/strategy.go

```diff
@@ -353,12 +355,34 @@ func getMonitorNamespace(clientset k8coresv1.CoreV1Interface, config *operatorut
 }
 
 func DumpInstallStrategyToConfigMap(clientset kubecli.KubevirtClient, operatorNamespace string) error {
+	// Clean up existing install strategy ConfigMaps
+	labelSelector := labels.NewSelector()
+	requirement, err := labels.NewRequirement(v1.InstallStrategyLabel, selection.Exists, nil)
+	if err != nil {
+		return fmt.Errorf("failed to create label selector requirement: %v", err)
+	}
+	labelSelector = labelSelector.Add(*requirement)
 
 	config, err := operatorutil.GetConfigFromEnv()
 	if err != nil {
 		return err
 	}
 
+	existing, err := clientset.CoreV1().ConfigMaps(config.GetNamespace()).List(context.Background(), metav1.ListOptions{
+		LabelSelector: labelSelector.String(),
+	})
+	if err != nil {
+		return fmt.Errorf("failed to list old install strategy configmaps: %v", err)
+	}
+
+	for _, cm := range existing.Items {
+		err := clientset.CoreV1().ConfigMaps(config.GetNamespace()).Delete(context.Background(), cm.Name, metav1.DeleteOptions{})
+		if err != nil {
+			return fmt.Errorf("failed to delete old install strategy configmap %s: %v", cm.Name, err)
+		}
+	}
```

Comment:
instead of listing all of the install strategy config maps, i think it is possible to delete all of them in one API call, without iterating the list and delete one object at a time, by using the `DeleteCollection` method.
could you please check?

Reply:
I wasn't aware of DeleteCollection. Thanks for pointing that out!